recipe/meta.yaml

```diff
@@ -1,196 +1,24 @@
-# Note, you cannot build this with macOS SDK < 10.12, despite there being a few patches
-# to add back support. QtWebengine scuppers us with:
-#   error: unknown type name 'SecKeyAlgorithm'
-# .. these 10.11 support patches should be removed anyway as they could break some things.
-# .. such as pop-up Window mouse-events.
-{% set version = "5.12.9" %}
+# This has become a metapacakge for qt-base and other related packages
+{% set version = "5.15.2" %}
```

Comment:
Why 5.15.2 when there's 5.15.3?

Reply:
we need to finish the icu70 migration correctly before we can do 5.15.3